src/routes/devices.ts

```diff
@@ -626,16 +626,16 @@ const upsertImageInstall = async (
 	releaseId: number,
 	dlProg?: number,
 ): Promise<void> => {
-	const [imgInstall] = (await resinApi.get({
+	const imgInstall = (await resinApi.get({
 		resource: 'image_install',
+		id: {
+			installs__image: imageId,
```

Comment:
BTW, will pine error if we specify an incomplete natural key or props that non-unique props in the `id` field?
Eg: just the `id: {device}`, or `id: {device, created_at}`

Reply:
Yes

Reply:
Awesome?